cmd/soroban-rpc/README.md

```diff
@@ -0,0 +1,62 @@
+# Soroban-RPC
+
+Soroban-RPC allows you to communicate directly with Soroban via a JSON RPC interface.
+
+For example, you can build an application and have it send a transaction, get ledger and event data or simulate transactions.
+
+## Dependencies
+  - [Git](https://git-scm.com/downloads)
+  - [Go](https://golang.org/doc/install)
+  - [Rust](https://www.rust-lang.org/tools/install)
+  - [Cargo](https://doc.rust-lang.org/cargo/getting-started/installation.html)
+
+## Building Stellar-Core
+Soroban-RPC requires an instance of stellar-core binary on the same host. This is referred to as the `Captive Core`. 
+Since, we are building RPC from source, we recommend considering two approaches to get the stellar-core binary:
+- If saving time is top priority and your development machine is on a linux debian OS, then consider installing the 
+testnet release candidates from the [testing repository.](https://apt.stellar.org/pool/unstable/s/stellar-core/)
+- The recommended option is to compile the core source directly on your machine:
+    - Clone the stellar-core repo:
+        ```bash
+        git clone https://github.com/stellar/stellar-core.git
+        cd stellar-core
+        ```
+    - Fetch the tags and checkout the testnet release tag:
+        ```bash
+        git fetch --tags
+        git checkout tags/v20.0.0-rc.2.1 -b soroban-testnet-release
+        ```
+    - Follow the build steps listed in [INSTALL.md](https://github.com/stellar/stellar-core/blob/master/INSTALL.md) file for the instructions on building the local binary
+
+## Building Soroban-RPC
+- Similar to stellar-core, we will clone the soroban-tools repo and checkout the testnet release tag:
+```bash
+git clone https://github.com/stellar/soroban-tools.git
+cd soroban-tools
+git fetch --tags
+git checkout tags/v20.0.0-rc4 -b soroban-testnet-release
+```
+- Install the required go packages:
+```bash
+make install
```

Comment:
you don't need to run `make install` that target will build and install cli to host machine's rust runtime path and rpc to go path